packages/atomic/src/components/atomic-layout/atomic-layout.pcss

```diff
@@ -0,0 +1,27 @@
+@import './search-layout.pcss';
```

Comment:
If I understand correctly, we will eventually import all layouts heres (commerce, service, etc), and then the "correct" one will be applied with the layout name.

But all styling for all layouts will always be downloaded by the end user ?

Reply:
We could go the separate component's route to scope that out

Reply:
@olamothe @btaillon perhaps we should relaunch that discussion. Having a separate layout component would only the necessary styles but lose the possibility of quickly switching style with the prop. Keeping it as is (should stay very light has it's simply layout but it scales a lot it might become an issue) would be a big heavier, or perhaps we can dynamically load the styles in another manner than the way Stencil does

Reply:
I'm fine with losing the quick layout switching, I don't think it's worth having to include unused CSS or having to find a workaround.

Reply:
Yeah, it's not mandatory that we can "insta switch" layout without reloading. So it's fine to go with a per component approach